src/test/java/tictactoe_game/PutMethodTest.java

```diff
@@ -8,255 +8,213 @@
+        { "EMPTY", "EMPTY", "EMPTY" } };
+
+    Grid newGrid = new Grid(emptyArray);
+
+    assertEquals(Grid.emptyGrid(), newGrid);
+  }
+
+  @Test
+  @DisplayName("Check if 'X' is on coordinate '0,0' after we `put` it on grid")
+  public void must_be_x_on_coordinate_0_0() {
+    Coordinate coord = new Coordinate(0, 0);
+    String[][] newGrid = { { "X", "EMPTY", "EMPTY" }, { "EMPTY", "EMPTY", "EMPTY" }, { "EMPTY", "EMPTY", "EMPTY" } };
+    Grid emptyGrid = Grid.emptyGrid();
+    Grid expectGrid = new Grid(newGrid);
+
+    assertEquals(Main.put(Element.X, emptyGrid, coord), expectGrid);
+  }
+
+  @Test
+  @DisplayName("Check if 'X' is on coordinate '0,1' after we `put` it on grid")
+  public void must_be_x_on_coordinate_0_1() {
+    Coordinate coord = new Coordinate(0, 1);
+    String[][] newGrid = { { "EMPTY", "X", "EMPTY" }, { "EMPTY", "EMPTY", "EMPTY" }, { "EMPTY", "EMPTY", "EMPTY" } };
+    Grid emptyGrid = Grid.emptyGrid();
+    Grid expectGrid = new Grid(newGrid);
+
+    assertEquals(Main.put(Element.X, emptyGrid, coord), expectGrid);
+  }
+
+  @Test
+  @DisplayName("Check if 'X' is on coordinate '0,2' after we `put` it on grid")
+  public void must_be_x_on_coordinate_0_2() {
+    Coordinate coord = new Coordinate(0, 2);
+    String[][] newGrid = { { "EMPTY", "EMPTY", "X" }, { "EMPTY", "EMPTY", "EMPTY" }, { "EMPTY", "EMPTY", "EMPTY" } };
+    Grid emptyGrid = Grid.emptyGrid();
+    Grid expectGrid = new Grid(newGrid);
+
+    assertEquals(Main.put(Element.X, emptyGrid, coord), expectGrid);
+  }
+
+  @Test
+  @DisplayName("Check if 'X' is on coordinate '1,0' after we `put` it on grid")
+  public void must_be_x_on_coordinate_1_0() {
+    Coordinate coord = new Coordinate(1, 0);
+    String[][] newGrid = { { "EMPTY", "EMPTY", "EMPTY" }, { "X", "EMPTY", "EMPTY" }, { "EMPTY", "EMPTY", "EMPTY" } };
+    Grid emptyGrid = Grid.emptyGrid();
+    Grid expectGrid = new Grid(newGrid);
+
+    assertEquals(Main.put(Element.X, emptyGrid, coord), expectGrid);
+  }
+
+  @Test
+  @DisplayName("Check if 'X' is on coordinate '1,1' after we `put` it on grid")
+  public void must_be_x_on_coordinate_1_1() {
+    Coordinate coord = new Coordinate(1, 1);
+    String[][] newGrid = { { "EMPTY", "EMPTY", "EMPTY" }, { "EMPTY", "X", "EMPTY" }, { "EMPTY", "EMPTY", "EMPTY" } };
+    Grid emptyGrid = Grid.emptyGrid();
+    Grid expectGrid = new Grid(newGrid);
+
+    assertEquals(Main.put(Element.X, emptyGrid, coord), expectGrid);
+  }
+
+  @Test
+  @DisplayName("Check if 'X' is on coordinate '1,2' after we `put` it on grid")
+  public void must_be_x_on_coordinate_1_2() {
+    Coordinate coord = new Coordinate(1, 2);
+    String[][] newGrid = { { "EMPTY", "EMPTY", "EMPTY" }, { "EMPTY", "EMPTY", "X" }, { "EMPTY", "EMPTY", "EMPTY" } };
+    Grid emptyGrid = Grid.emptyGrid();
+    Grid expectGrid = new Grid(newGrid);
+
+    assertEquals(Main.put(Element.X, emptyGrid, coord), expectGrid);
```

Comment:
Please, change the arguments to comply with method semantics /expected value first, then the actual one/.